geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -1625,10 +1629,10 @@ public boolean invalidate(EntryEventImpl event, boolean invokeCallbacks, boolean
                       if (isDebugEnabled) {
                         logger.debug("Invalidate: Entry already invalid: '{}'", event.getKey());
                       }
-                      if (event.getVersionTag() != null && owner.getVersionVector() != null) {
-                        owner.getVersionVector().recordVersion(
-                            (InternalDistributedMember) event.getDistributedMember(),
-                            event.getVersionTag());
+                      processVersionTag(re, event);
```

Comment:
processVersionTag can throw ConcurrentModificationException but I do not see that it will be caught here. But I also don't see it handled in the existing code on line 1384. Do we somehow know in these contexts that it will never throw?

Reply:
disregard this comment; I found were it handles that exception (line: 1510)